Tests/NetworkRequestTests.swift

```diff
@@ -104,4 +104,73 @@ class NetworkRequestTests: XCTestCase {
         XCTAssertEqual(urlRequest.cachePolicy, cachePolicy, file: file, line: line)
         XCTAssertEqual(urlRequest.timeoutInterval, timeout, file: file, line: line)
     }
+    
+//    func test_NetworkRequestQueryParameterEncodingStrategy_EncodeUrlQueryAllowedCharacterSet() {
```

Comment:
Probably want to remove commented out code?

Reply:
Removing